atmo/templates/atmo/new-cluster.jinja

```diff
@@ -33,6 +33,13 @@
           <p class="help-block"></p>
         </div>
         <div class="alert alert-warning">Upload your SSH <strong>public key</strong>, not private key! This will generally be found in places like <code>~/.ssh/id_rsa.pub</code>.</div>
+        <div class="form-group">
+          <label>EMR release</label>
+          <div class="control-group">
+            {{ new_cluster_form.emr_release }}
```

Comment:
I would rewrite the whole form template tbh. There are a few things there that break the DRY principle at the moment. Let's talk about it.